### DISCUSSION TITLE
Why are PraisonAI agents not supported in the GO version?

### DISCUSSION THREAD

**Discussion by fonteijne** [danielmiessler/Fabric#934] [Q&A] [answered]
I just updated to the go version, but now the `-a` flag is unknown. 
Why is this feature dropped? Or is it waiting to be integrated?

**Comment by xssdoctor** [comment, marked as answer]
The honest answer is that I didn't think it was a good feature. Were you able to make anything useful with it?

**Reply by fonteijne** [reply]
Well, It's still really new for me. I started learning about the agents and noticed Fabric supported PraisonAI. Therefore I took some time to learn PraisonAI, only to find out the support is dropped. Bad luck, haha. 

So, to answer your question; No (not yet). 
But I do think, if you were to support/ integrate agents again; a low-code solution would fit best.

**Comment by ajoslin103** [comment]
This makes me so sad.  I'd made it my goal to write and deploy an Agent from Fabric by Halloween

I've converted all my machines to the Go version & now I have to go back to Python to the AtheeSouhait fork, as the only way to use Fabric at work is with the Azure OpenAI -- argh, and I lost my .env !!  I'll come back with that when I get it working again...

Can we start an effort to bring the Agent support to the Go version?  If anyone [@danielmiessler?] could point me to where the Agent code was in the Python I'd at least start looking at it...  

ref: 
   https://github.com/danielmiessler/fabric/issues/179
   you must clone his entire repo and force a new install of fabric using pipx install . --force (revs on deps ruins it)
   ref: https://github.com/danielmiessler/fabric/issues/179#issuecomment-2184015595
